docs/running-deploying.md

```diff
@@ -28,13 +28,22 @@ make install-requirements
 
 ## Running the Simulator Locally
 
-Before running the Azure OpenAI API Simulator you should ensure that you have set up your local config. See [Azure OpenAI API Simulator Configuration Options](./docs/config.md) for details on how to do this.
-
-To run the simulated API, use the following command from the repository root directory:
-
-```console
-make run-simulated-api
-```
+1. Before running the Azure OpenAI API Simulator you should ensure that you have set up your local config. See [Azure OpenAI API Simulator Configuration Options](./config.md) for details on how to do this. The first time you run this, you should be sure to set at least the following environment variables.
+
+    ```python
+    SIMULATOR_API_KEY=my-test-key
+    TEST_OPENAI_ENDPOINT=http://localhost:8000/
+    TEST_OPENAI_KEY=my-test-key
+    ```  
+
+2. Start (or restart) the simulator by running the following command in your terminal from the repository root directory:
```

Comment:
What do you mean by restart? If you run `make run-simulated-api` it doesn't return until the service is killed, and if I open another terminal and run `make run-simulated-api` a second time it won't work.

I assume you're saying "or stop, and then restart it, if it's already running" but I think we should assume that anyone going through this "first run" experience won't have started it yet. You can't run the `make` command if it's already running, so anyone following these 4 steps will have to stop it to follow step 2.

Reply:
Updated with another line regarding restarting the server to pick up config changes.